pkg/controller/kubelet-config/kubelet_config_nodes.go

```diff
@@ -76,6 +76,12 @@ func (ctrl *Controller) syncNodeConfigHandler(key string) error {
 	if err := ctrl.cleanUpDuplicatedMC(managedNodeConfigKeyPrefix); err != nil {
 		return err
 	}
+	// explicitly setting the cgroupMode to "v2" and also updating the config node's spec if found empty
+	// This helps in updating the cgroupMode on all the worker nodes if they still have cgroupsv1 (Ex: RHEL8 workers)
+	if nodeConfig.Spec.CgroupMode == emptyInput {
+		nodeConfig.Spec.CgroupMode = osev1.CgroupModeDefault
```

Comment:
TRT team has asked us to have one PR that updates the cgroup to v2. We can cleanup the API pieces later. For now, can this be set to 
```
nodeConfig.Spec.CgroupMode = osev1.CgroupModeV2
```
basically, ignorign CgroupModeDefault?

Reply:
Sure, Addressed

Reply:
> TRT team has asked us to have one PR that updates the cgroup to v2. We can cleanup the API pieces later. For now, can this be set to
> 
> ```
> nodeConfig.Spec.CgroupMode = osev1.CgroupModeV2
> ```
> 
> basically, ignorign CgroupModeDefault?

/hold cancel